nrf_modem/doc/api.rst

```diff
@@ -268,6 +268,8 @@ A-GNSS data request bitmask values
    :project: nrfxlib
    :members:
 
+.. _agnss_data_type_enum:
```

Comment:
```suggestion
.. _agnss_data_type_enum_api:
```
The link is mentioned as above here: https://github.com/nrfconnect/sdk-nrf/pull/14859/files#diff-478a3860a34e7a4faf66f620bee86e14a1fd4b71e4e179908370313fa5703506R135